components/dropdown/dropdown-content-styles.js

```diff
@@ -61,16 +64,26 @@ export const dropdownContentStyles = css`
 		top: -7px;
 		z-index: 1;
 	}
+	:host([align="start"]) .d2l-dropdown-content-pointer,
+	:host([align="end"][dir="rtl"]) .d2l-dropdown-content-pointer {
+		left: min(calc(1rem + ${(pointerRotatedLength - pointerLength) / 2}px), calc(50% - ${pointerLength / 2}px)); /* 1rem corresponds to .d2l-dropdown-content-container padding */
+		right: auto;
+	}
+	:host([align="end"]) .d2l-dropdown-content-pointer,
+	:host([align="start"][dir="rtl"]) .d2l-dropdown-content-pointer {
+		left: auto;
+		right: min(calc(1rem + ${(pointerRotatedLength - pointerLength) / 2}px), calc(50% - ${pointerLength / 2}px)); /* 1rem corresponds to .d2l-dropdown-content-container padding */
+	}
```

Comment:
The pointer positioning previously was always approximately centre with the opener, which is not correct for the cases with align values and wide openers.
I checked with Jeff and he wanted it to be the same behaviour as tooltip, which is what this is.

Note that in tooltip this has a backup for browser that don't support `min`. I checked our updated supported browsers chart and compared that with caniuse and all cases where browsers don't support it correspond with our unsupported browsers. Let me know if I should still add a backup just in case.